### DISCUSSION TITLE
This isn't an issue, but a compliment! Incredible detail!!! April fools!!!

### DISCUSSION THREAD

**Discussion by sunfloweros** [ftde0/yt2009#189] [General]
I self host a yt-2009 instance, today is April 1 AKA April Fools. You guys programmed the upside down flip into the website! You recreated the joke from Google in 2009, THAT'S AWESOME! Thanks for the great work!

**Comment by ftde0** [comment]
thanks for the great words! :D 

lmk if you've got any questions/issues, otherwise feel free to close as completed
